nixos/lib/make-options-doc/options-to-docbook.xsl

```diff
@@ -20,7 +20,19 @@
       <title>Configuration Options</title>
       <variablelist xml:id="configuration-variable-list">
         <xsl:for-each select="attrs">
-          <xsl:variable name="id" select="concat('opt-', str:replace(str:replace(str:replace(str:replace(attr[@name = 'name']/string/@value, '*', '_'), '&lt;', '_'), '>', '_'), ':', '_'))" />
+          <xsl:variable name="id" select="
+            concat('opt-',
+              str:replace(str:replace(str:replace(str:replace(str:replace(str:replace(str:replace(
+                attr[@name = 'name']/string/@value,
+                '*', '_'),
+                '&lt;', '_'),
+                ' ', '_'),
+                '>', '_'),
+                '[', '_'),
+                ']', '_'),
+                ':', '_')
+              )
+          " />
```

Comment:
```suggestion
          <xsl:variable name="id" select="concat('opt-', translate(attr[@name = 'name']/string/@value, '*&lt; >[]:', '_______'))" />
```

Reply:
Applied and checked some ids: :heavy_check_mark: